bin/demultiplexing.sh

```diff
@@ -173,7 +173,8 @@ log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Log files will be written
 # Looping through sub dirs to see if all files.
 #
 log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "ls -1 -d ${SEQ_DIR}/*/"
-for i in $(ls -1 -d "${SEQ_DIR}/"*/)
+projects=$(find "${SEQ_DIR}/" -type d)
+for i in "${projects}"[@]
```

Comment:
Dit
`for i in "${projects}"[@]`
gaat niet werken. Zal eerder dit moeten zijn:
`for i in "${projects[@]}"`